benchmarks/transformer/score_mod.py

```diff
@@ -156,27 +157,53 @@ def eager_sdpa(query, key, value, _):
 
     if mask_mod:
         block_mask = create_block_mask(
-            mask_mod, 1, 1, q_seq_len * (q_heads // kv_heads), kv_seq_len, query.device
+            mask_mod, 1, 1, q_seq_len, kv_seq_len, query.device
         )
     else:
         block_mask = _create_empty_block_mask(query, key)
 
+    if mask_mod:
+        attn_mask = create_mask(mask_mod, 1, 1, query.shape[-2], key.shape[-2])
```

Comment:
The SDPA PR should have landed so you should be able to just call sdpa with enable_gqa similiar to flex